### ISSUE TITLE
[Navigate to and select "Input.Date" list item in "Inputs" section]: Voice Access is not able to click 'Show Next Month' and 'Show Previous Month' buttons under 'Show Date Picker' button

### ISSUE THREAD

**Issue by Shireen2000** [microsoft/AdaptiveCards#7507]
### Target Platforms

Other

### SDK Version

103.0.1255.0 

### Application Name

Adaptive Cards

### Problem Description

https://microsoft.visualstudio.com/OS/_queries/edit/39920618/?triage=true

Test Environment:
OS: 22H2 (OS Build 25120.1000)
Browser: Edge Dev
Browser Version: Version 103.0.1255.0 (Official build) dev (64-bit)
URL: [Adaptive Cards](https://adaptivecards.io/)
Tool: Voice Access

Repro Steps:
Launch the above URL.
Command: "Voice access wake up"
Command: "Click Schema Explorer" for the top header.
Command: 'Click Input.Date' from under 'Inputs' label.
Navigate till the first card present under this page.
Command: 'Click Show date Picker.
Date Picker will get open.
Command: 'Click Next Month' or 'Show Previous Month'.
Observe the issue.
Actual Result:
Voice Access is not able to click 'Show Next Month' and 'Show Previous Month' buttons under 'Show Date Picker' button.
However, we are able to click 'Show Next Month' and 'Show Previous Month' buttons using Grid view.

Expected Result:
Voice Access users should be able to click 'Show Next Month' and 'Show Previous Month' buttons under 'Show Date Picker' button.

Note: This issue is repro throughout this page.

Observation: 'IsKeyboardFocusable' property is set as 'True' for both 'Show Next Month' and 'Show Previous Month' buttons.

User Experience:

People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access to use OneNote may not be able edit or type they will find difficulty in accessing 'Canvas area' and 'Canvas Title' edit field. Voice access users will get impact and will not able to interact with controls with pre-defined Voice access commands.
 
MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)
 
Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

To learn more about Voice Access and how an application can support its different modes, Refer: [VoiceAccess Testing - OSGWiki](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.osgwiki.com%2fwiki%2fVoiceAccess_Testing&data=05%7c01%7cv-shiwasi%40microsoft.com%7c00297be07a15467b634408da325a247c%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637877659989570305%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=sQj%2Bb4UzcB2jUBsX3r%2f4%2BjkjzUrXncXKGhGCIuu8vc4%3D&reserved=0)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @Shireen2000!

I've done some investigating here, and I believe we should close this bug. For our `Input.Date` control, we utilize HTML's `input type="date"`. With this implementation, the browser has control over the element as you can see in the browser comparisons below:

**Edge**
![Screenshot (1)](https://user-images.githubusercontent.com/98650930/194956852-0f7c17a7-c20c-4a46-936b-bb138adb0c11.png)

**Firefox**
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/98650930/194956937-fe5d9056-4e36-4b29-b602-3f252366dd14.png)

Because of this, we cannot update the behavior/name of the `Show next month` and `Show previous month` buttons.

However, without making changes, we were able to navigate the control via voice access using "Tab" and "Enter" to tab through the element and select the correct button.

I believe using "Tab" and "Enter" commands to access the control meets the accessibility requirements, and we should be able to close this issue.

Please let me know what you think and if you have any questions for me. Thanks!
